### ISSUE TITLE
Bug Report: Alt note doesn't work

### ISSUE THREAD

**Issue by HumanKR** [nebulazorua/andromeda-engine#105]
### If you are playing a downloaded version of the game, what operating system are you using?
Windows (`x86`), Windows (`x86_64`), Linux, or macOS? Specify below.

Windows 10 (`x86_64`)

## What version of the game are you using? Look in the bottom left corner of the main menu.
latest version

## Have you identified any steps to reproduce the bug? If so, please describe them below. Use images if possible.

Actually I'm not good at english sooo i'll try to explain this bug in a nutshell.
![image](https://user-images.githubusercontent.com/45139720/175614604-d9944180-0bb8-4877-828d-d6eaeed2e031.png)

in the playstate, there is some code about altanim inside playAnimationNote function.
ok if noteType is alt, then altAnim is '-alt' right?
then lets go to that function.

![image](https://user-images.githubusercontent.com/45139720/175614885-04497107-ff5b-48f8-b76b-4ac3a3636625.png)

and this is important part of this function about alt animation.
i add some trace for figure out what is problem.

![image](https://user-images.githubusercontent.com/45139720/175615140-31e1c4a9-4328-461c-bc7f-09a00e036517.png)

yea... the final suffix is "-alt-alt"
we must be delete the code `if(daNote.noteType=='alt')suffix+='-alt';`
becuase that's already got suffix before
(`altAnim = '-alt';` in `!daNote.mustPress && daNote.canBeHit && !daNote.wasGoodHit`)

![image](https://user-images.githubusercontent.com/45139720/175615842-35c97428-0603-447b-b6ed-1ff82c67ea01.png)

aaaand yea it works nicely

If you have any other good ideas, please fix them <3

**Comment by nebulazorua** [comment]
yeah i noticed this a while ago and I've been meaning to get around to fixing this, but I've been busy w/ alot of projects
If you'd like to do a pull request w/ your fix I'll merge it

**Comment by HumanKR** [comment]
> yeah i noticed this a while ago and I've been meaning to get around to fixing this, but I've been busy w/ alot of projects If you'd like to do a pull request w/ your fix I'll merge it

I pulled!
I don't know because it's my first time, but I think it went well!
